bp_common/test/Makefile.frag

```diff
@@ -465,20 +468,21 @@ BEEBS_TESTS = \
   ud \
   whetstone \
   wikisort
-  # qsort works, but there's a name conflict with riscv-tests. We should fix this
+  # qsort works, but there's a name conflict with riscv_tests. We should fix this
   #   by putting each test suite in its own mem directory
```

Comment:
Does this PR fix this name conflict, or is it still present?

Reply:
fixed! I'll remove the comment